apps/ledger-live-desktop/src/renderer/actions/settings.ts

```diff
@@ -441,3 +442,10 @@ export const setMevProtection = (payload: boolean) => ({
   type: "SET_MEV_PROTECTION",
   payload,
 });
+
+export const toggleShouldDisplayMemoTagInfo = (payload: boolean) => {
+  return {
+    type: SETTING_TOGGLE_MEMOTAG_INFO,
```

Comment:
Why don't do like `SET_MEV_PROTECTION` & others?

Reply:
so i can change the name of this constant, but it's value is recommended by the naming convention of redux action https://redux.js.org/style-guide/#model-actions-as-events-not-setters
I don't usually use GETTERs & SETTERs as action types.
I'll change the name

Reply:
edited to an action based name `TOGGLE_MEMOTAG_INFO`

Reply:
I was thinking more about where you put this constant :) 

Why don't use the same file as others?

Reply:
all LLD action types are string hardcoded (including `SET_MEV_PROTECTION`)  in the action creators here -> `apps/ledger-live-desktop/src/renderer/actions/settings.ts`, inlike the ledger-live-mobile where all actions are in an enum. So i created a constants (that can be an enum also) file that will regroup all action types there